jib-cli/src/test/java/com/google/cloud/tools/jib/cli/jar/JarFilesTest.java

```diff
@@ -146,4 +151,222 @@ public void testToJibContainerBuilder_packagedStandard_basicInfo()
                 .build()
                 .getEntries());
   }
+
+  @Test
+  public void testToJibContainerBuilder_explodedLayeredSpringBoot_basicInfo()
+      throws IOException, URISyntaxException, InvalidImageReferenceException {
+    Path springbootJar = Paths.get(Resources.getResource(SIMPLE_SPRING_BOOT_LAYERED).toURI());
+    Path destDir = temporaryFolder.getRoot().toPath();
+    JibContainerBuilder containerBuilder =
+        JarFiles.toJibContainerBuilder(springbootJar, destDir, ProcessingMode.exploded);
+    ContainerBuildPlan buildPlan = containerBuilder.toContainerBuildPlan();
+
+    assertThat(buildPlan.getBaseImage()).isEqualTo("gcr.io/distroless/java");
+    assertThat(buildPlan.getPlatforms()).isEqualTo(ImmutableSet.of(new Platform("amd64", "linux")));
+    assertThat(buildPlan.getCreationTime()).isEqualTo(Instant.EPOCH);
+    assertThat(buildPlan.getFormat()).isEqualTo(ImageFormat.Docker);
+    assertThat(buildPlan.getEnvironment()).isEmpty();
+    assertThat(buildPlan.getLabels()).isEmpty();
+    assertThat(buildPlan.getVolumes()).isEmpty();
+    assertThat(buildPlan.getExposedPorts()).isEmpty();
+    assertThat(buildPlan.getUser()).isNull();
+    assertThat(buildPlan.getWorkingDirectory()).isNull();
+    assertThat(buildPlan.getEntrypoint())
+        .isEqualTo(
+            ImmutableList.of("java", "-cp", "/app", "org.springframework.boot.loader.JarLauncher"));
+    assertThat(buildPlan.getLayers().size()).isEqualTo(4);
```

Comment:
Can we also check the name of each layer?

Reply:
Done.